includes/class-omise-payment-result.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @since 4.0
+ */
+class Omise_Payment_Result {
```

Comment:
Note: still 50/50 of this classname. Should it be `Omise_Order_Result` instead? Since this class is mainly designed to be used for resolving a payment and properly updating a particular order status accordingly.